### PR TITLE
💄 Updated OG Image

### PR DESCRIPTION
**Pull request by joshbermanssw**
As per email Re: Tina.io open graph OG
cc: @adamcogan @betty

We have upgraded the OG image

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ const MainLayout = ({ Component, pageProps }) => {
           site_name: data.title,
           images: [
             {
-              url: 'https://tinacms.org/img/tina-twitter-share.png',
+              url: './public/img/tina-og.png',
```
Comment (JackDevAU) [inline pages/_app.tsx:48]: does this need the `./public` ?

File: pages/_app.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ const MainLayout = ({ Component, pageProps }) => {
           site_name: data.title,
           images: [
             {
-              url: 'https://tinacms.org/img/tina-twitter-share.png',
```
Comment (bradystroud) [inline pages/_app.tsx:48]: Is the old image stored on this repo? If so you can remove it :)

Comment (joshbermanssw) [reply pages/_app.tsx:48]: ✅

File: pages/_app.tsx (hunk 1 above)
Comment (joshbermanssw) [reply pages/_app.tsx:48]: i would imagine so

Comment (JackDevAU) [reply pages/_app.tsx:48]: https://www.opengraph.xyz/url/https%3A%2F%2Ftina-jx24qfqr2-tinacms.vercel.app%2F

Seems like somethings wrong

Comment (joshbermanssw) [reply pages/_app.tsx]: yes - was using path dir not url

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**